conf/application.conf

```diff
@@ -82,5 +86,6 @@ tracking-consent-frontend {
 }
 
 features {
-  welsh-translation: true
+  welsh-translation = true
+  dstNewRegistrationFrontendEnable = false
```

Comment:
this needs to be true. I believe it should only be false for production config

Reply:
ill updated this while testing i kept it false, and missed to change it back.